src/applications/mhv-landing-page/tests/containers/WelcomeContainer.unit.spec.jsx

```diff
@@ -66,4 +51,39 @@ describe('WelcomeContainer component', () => {
     const { getByRole } = setup(initialState);
     getByRole('heading', { name: 'Welcome' });
   });
+
+  describe('Profile links', () => {
+    it('calls datadogRum.addAction on click of profile links', async () => {
+      const { getByRole } = setup();
+      const spyDog = sinon.spy(datadogRum, 'addAction');
+      const profileLink = getByRole('link', { name: /profile/i });
+      // Change the link to an anchor, so JSDOM does not complain about navigation
+      profileLink.href = '#dummy-link';
+      fireEvent.click(profileLink);
+
+      await waitFor(() => {
+        expect(spyDog.called).to.be.true;
+      });
+
+      spyDog.restore();
+    });
+
+    it('reports to GA on click of profile links', async () => {
+      const eventData = {
+        event: 'nav-linkslist',
+        'links-list-header': 'Profile',
+        'links-list-section-header': 'Welcome',
+      };
+      global.window.dataLayer = [];
```

Comment:
Test was not able to detect the ga even when I used `const spyRecordEvent = sinon.spy()` - so found how other teams are using `global.window.dataLayer`  to test GA events